Driver/ZendLdapDriver.php

```diff
@@ -86,6 +86,9 @@ public function bind(UserInterface $user, string $password): bool
 
     /**
      * Treat a Zend Ldap Exception.
+     *
```

Comment:
Please undo the changes on this file and then it's ok.

Reply:
Sorry I was on hollidays. I haven't read it until today. I'll try do what you say.

Reply:
I think it's done.